documentation/Contributing-Development/Docs/Setup.md

```diff
@@ -7,7 +7,7 @@ your browser. The [Guidelines](Guidelines.md) still apply!
 
 All the files are markdown files. Markdown is the stuff you partially know from discord text highlighting.
 
-`**Bold text**`  **result**    
+`**Bold text**`  **result**  
 `_Italic text_` _result_
```

Comment:
Why delete just two spaces? `` `_Italic text_` _result_`` doesn't have any trailing space.

Reply:
two spaces make a line break. 4 spaces do not have more effect